pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/extensions/ExtensibleLoadManagerImpl.java

```diff
@@ -874,12 +876,20 @@ public List<Metrics> getMetrics() {
         }
 
         metricsCollection.addAll(this.assignCounter.toMetrics(pulsar.getAdvertisedAddress()));
-
         metricsCollection.addAll(this.serviceUnitStateChannel.getMetrics());
+        metricsCollection.addAll(toMetrics(pulsar.getAdvertisedAddress()));
 
         return metricsCollection;
     }
 
+    private List<Metrics> toMetrics(String advertisedBrokerAddress) {
+        var dimensions = Map.of("broker", advertisedBrokerAddress, "metric", "bundleReleasing");
```

Comment:
Should we label "bundleUnloading" for its metric name, similar to other unload metrics?